src/python/WMCore/WMSpec/WMWorkloadTools.py

```diff
@@ -153,13 +127,18 @@ def _validateArgumentOptions(arguments, argumentDefinition, optionKey):
 
 
 def _validateInputDataset(arguments):
+    
     inputdataset = arguments.get("InputDataset", None)
     dbsURL = arguments.get("DbsUrl", None)
     if inputdataset != None and dbsURL != None:
-        result = _verifyDBSCall(dbsURL, "datasets?&dataset_access_type=*&dataset=%s" % inputdataset)
-        if len(result) == 0:
-            msg = "Inputdataset %s doesn't exist on %s" % (inputdataset, dbsURL)
-            raise WMSpecFactoryException(msg)
+        #import DBS3Reader here, since Runtime code import this module and worker node doesn't have dbs3 client 
+        from WMCore.Services.DBS.DBS3Reader import DBS3Reader
+        from WMCore.Services.DBS.DBSErrors import DBSReaderError
+        try:
```

Comment:
I'm not saying we should, but how about we then create a method in this file that interacts with DBS server?
We can have the specific import in a single place and handle the api to be called from there.